java/src/org/openqa/selenium/chromium/AddHasFedCm.java

```diff
@@ -0,0 +1,98 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.chromium;
```

Comment:
Since this is based on "Federated Credential Management API", the package name shouldn't be `chromium`, but probably something parallel to the `virtualauthenticator`. Perhaps `org.openqa.selenium.federatedcredentialmanagement`?